### ISSUE TITLE
CI failure

### ISSUE THREAD

**Issue by treywelsh** [OpenNebula/terraform-provider-opennebula#60]
It seems that the CI of the repository doesn't work anymore.
Here are some filtered logs.
```
$ rvmsudo /usr/share/one/install_gems --yes
...
Using nokogiri 1.10.9
...
Bundle complete! 44 Gemfile dependencies, 95 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed
$ source .travis/before_script/oned.sh
Ignoring augeas-0.6.4 because its extensions are not built. Try: gem pristine augeas --version 0.6.4
...
Ignoring nokogiri-1.10.9 because its extensions are not built. Try: gem pristine nokogiri --version 1.10.9
...
Error: nokogiri gem not installed.
The command "source .travis/before_script/oned.sh" failed and exited with 255 during .
```

Full logs are here: https://travis-ci.org/github/OpenNebula/terraform-provider-opennebula/builds/733708888

**Comment by treywelsh** [comment]
If you have some time to take a look @christian7007 

Thank you !

**Comment by treywelsh** [comment]
Fixed by #70
